vMenu/menus/VehicleOptions.cs

```diff
@@ -1090,6 +1092,23 @@ void HandleListIndexChanges(Menu sender, MenuListItem listItem, int oldIndex, in
                         // Close all doors.
                         SetVehicleDoorsShut(veh.Handle, false);
                     }
+
+                    // If the index >= 6, the bomb doors could be affected
+                    if (index >= 6)
+                    {
+
+                        bool bombBayOpen = CitizenFX.Core.Native.Function.Call<bool>(CitizenFX.Core.Native.Hash._ARE_BOMB_BAY_DOORS_OPEN, veh.Handle);
```

Comment:
I'm using the version which is included when you `git clone` this repo :)

Reply:
That's very strange since it's never been a void at all...?
https://github.com/citizenfx/natives/blob/50395415d68c6b9e7bb9424114979c6718dfd84e/VEHICLE/AreBombBayDoorsOpen.md

Reply:
It's definitely very odd, but that is the way it exists in the API for now, despite the natives database clearly showing it's a return type of `bool`. If they update the API we can change this to use the API in the future of course.

Reply:
I've changed it. Apparently it should've been 'BOOL' instead of 'bool', not sure why it was accepted when I first added those natives if the return type was obviously broken to the elements but oh well. That's now been taken care off. I'll build a new API dll by building FiveM and upload that really quickly.